ql/src/test/results/clientpositive/spark/timestamp_1.q.out

```diff
@@ -257,7 +329,7 @@ POSTHOOK: query: select cast(t as double) from timestamp_1 limit 1
 POSTHOOK: type: QUERY
 POSTHOOK: Input: default@timestamp_1
 #### A masked pattern was here ####
-1.2938436611E9
+1.293843661E9
```

Comment:
Why is this output changed? Apparently not in original patches

Reply:
This is because is oss master this test did not run in spark cli driver. This output was changed in llap therefore required in spark as well. This was done to make both q.out same.

![image](https://github.com/apache/hive/assets/104416558/fedbb4fc-a22a-4573-920f-ca4672896ce5)